src/lib/isPostalCode.js

```diff
@@ -28,6 +28,7 @@ const patterns = {
   HR: /^([1-5]\d{4}$)/,
   HU: fourDigit,
   ID: fiveDigit,
+  IE: /^[a-zA-Z0-9]{3}\b \b[a-zA-Z0-9]{4}$/i,
```

Comment:
Simplified regex would be:
```javascript
/^\w{3}\s\w{4}$/
```

`\w` will match any alphanumeric, see a good [quick ref here](https://regexone.com/).

However, it also looks to me like most of the areas, the `routing key` is in the format of `[alpha][digit][digit]` except for `Dublin 6W`, so we could make it more specific?
```javascript
/^[A-z][0-9][0-9|W]\s\w{4}$/
```

Reply:
Checking on this @profnandaa.

Reply:
@profnandaa, it should be
```javascript 
/^[A-z]\d[\d|w]\s\w{4}$/i
```
 Instead of 
```javascript 
/^[A-z]\d[\d|W]\s\w{4}$/i
```

Reply:
Also, `\s` will match tab and new line, 
Should we allow for tabs and a new line for separation?

Reply:
Sure, you we could just do an actual space.

Reply:
```javascript
/^[A-z]\d[\d|w] \w{4}$/
```